source/views/news/lib/util.ts

```diff
@@ -0,0 +1,19 @@
+import {decode} from '@frogpond/html-lib'
+import {toLaxTitleCase} from '@frogpond/titlecase'
+import {StoryType} from '../types'
+
+// remove all entries with blank excerpts
+// remove all entries with a <form from the list
+export const cleanEntries = (data: StoryType[]): StoryType[] => {
+	return data
+		.filter((entry) => entry.excerpt.trim() !== '')
+		.filter((entry) => !entry.content.includes('<form'))
+}
+
+// remove extraneous whitspace
+// decode entities
+// titlecase words
+export const trimStoryCateogry = (label: string): string => {
+	let evenedWhitespace = label.replace(/\s+/gu, ' ')
+	return toLaxTitleCase(decode(evenedWhitespace))
+}
```

Comment:
Alternatively we could:
- perform this cleanup on ccc
- provide an array of categories from the server response

Reply:
I am okay with continuing to keep this here, though!